Description:  Fix hardening
Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Forwarded: no

Index: ladspa-sdk/src/makefile
===================================================================
--- ladspa-sdk.orig/src/makefile	2013-06-15 12:05:32.948969369 +0200
+++ ladspa-sdk/src/makefile	2013-06-15 12:07:39.308974508 +0200
@@ -14,9 +14,10 @@
 #
 
 INCLUDES	=	-I.
-LIBRARIES	=	-ldl -lm -lc
-CFLAGS		=	$(INCLUDES) -Wall -Werror -O2 -fPIC
-CXXFLAGS	=	$(CFLAGS)
+LIBRARIES	=	-ldl -lm -lc -z relro -z now
+CFLAGS		=
+CXXFLAGS	=
+LDFLAGS		=
 PLUGINS		=	../plugins/amp.so				\
 			../plugins/delay.so				\
 			../plugins/filter.so				\
@@ -39,7 +40,7 @@
 
 ../plugins/%.so:	plugins/%.cpp ladspa.h
 	$(CPP) $(CXXFLAGS) -o plugins/$*.o -c plugins/$*.cpp
-	$(CPP) -o ../plugins/$*.so plugins/$*.o -shared
+	$(CPP) -o ../plugins/$*.so plugins/$*.o -shared $(LDFLAGS)
 
 ###############################################################################
 #
